src/django/iow/settings.py

```diff
@@ -14,6 +14,7 @@
 import requests
 
 from pathlib import Path
+from django.core.exceptions import ImproperlyConfigured
```

Comment:
Is this import used?

Reply:
Yes, it was used without importing it:

https://github.com/azavea/iow-boundary-tool/blob/5e23e9713eaa18953b5f99a318b832b2d5684e1b/src/django/iow/settings.py#L30-L32

https://github.com/azavea/iow-boundary-tool/blob/5e23e9713eaa18953b5f99a318b832b2d5684e1b/src/django/iow/settings.py#L64

https://github.com/azavea/iow-boundary-tool/blob/5e23e9713eaa18953b5f99a318b832b2d5684e1b/src/django/iow/settings.py#L69-L70

This adds the missing import.